src/wagtail_ai/views.py

```diff
@@ -79,8 +80,8 @@ def process(request):
     if not text:
         return JsonResponse(
             {
-                "error": "No text provided - please enter some text before using AI \
-                    features"
+                "error": "No text provided - "
+                "please enter some text before using AI features"
```

Comment:
**Nitpick:** Should the error message end with a full stop?

Reply:
done